src/components/PostDetails.tsx

```diff
@@ -140,7 +70,11 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() => {
+                      dispatch(commentActions
+                        .removeComment(comment.id));
+                      dispatch(commentActions.remove(comment.id));
+                    }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)